Header header
bool attached
std_msgs/String name
int32 serial
int32 device_class
int32 device_id
int32 version
